src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2276,6 +2276,482 @@ def save_lora_weights(
         )
 
 
+class CogVideoXLoraLoaderMixin(LoraBaseMixin):
```

Comment:
looks good! 
just wonder if flux Lora loader mixin work on cogvideox Lora out of box right now? (even if it does, I think it is a good idea to keep it as a separate class because we may need to add logics specific to them later)

Reply:
Yep, the Flux one works here too